src/hooks/useNotificationTypes.tsx

```diff
@@ -230,7 +230,8 @@ export const notificationTypes: NotificationTypeConfig[] = [
           });
 
           const toChainEta = status?.toChain?.chainData?.estimatedRouteDuration ?? 0;
-          const estimatedDuration = formatSeconds(Math.max(toChainEta, 0));
+          const estimatedDuration =
+            typeof toChainEta === 'string' ? toChainEta : formatSeconds(Math.max(toChainEta, 0));
```

Comment:
Can we rewrite this without a type check? I think I'd prefer we don't coalesce above then do if  != null.

Reply:
yeah same tbh. i'll rethink this a bit and see what we can get away with